charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,289 @@
+              bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+              kubernetes_sd_configs:
+                - role: node
+              relabel_configs:
+                - replacement: kubernetes.default.svc.cluster.local:443
+                  target_label: __address__
+                - regex: (.+)
+                  replacement: /api/v1/nodes/$${1}/proxy/metrics/cadvisor
+                  source_labels:
+                    - __meta_kubernetes_node_name
+                  target_label: __metrics_path__
+              scheme: https
+              tls_config:
+                ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                insecure_skip_verify: false
+                server_name: kubernetes
+            - job_name: kubelet
+              bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+              kubernetes_sd_configs:
+                - role: node
+              relabel_configs:
+                - replacement: kubernetes.default.svc.cluster.local:443
+                  target_label: __address__
+                - regex: (.+)
+                  replacement: /api/v1/nodes/$${1}/proxy/metrics
+                  source_labels:
+                    - __meta_kubernetes_node_name
+                  target_label: __metrics_path__
+              scheme: https
+              tls_config:
+                ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                insecure_skip_verify: false
+                server_name: kubernetes
+            # node_exporter job not needed, but leaving configuration in case users want to enable it
+            # - job_name: node_exporter
+            #   kubernetes_sd_configs:
+            #     - role: pod
+            #   relabel_configs:
+            #     - action: keep
+            #       regex: prometheus-node-exporter.*
+            #       source_labels:
+            #         - __meta_kubernetes_pod_label_app_kubernetes_io_name
+            #     - action: replace
+            #       source_labels:
+            #         - __meta_kubernetes_pod_node_name
+            #       target_label: instance
+            #     - action: replace
+            #       source_labels:
+            #         - __meta_kubernetes_namespace
+            #       target_label: namespace
+      filelog:
+        include:
+          - /var/log/pods/*/*/*.log
+          - /var/log/*/*
+          - /var/log/*
+          - /var/log/alternatives.log
+          - /var/log/cloud-init.log
+          - /var/log/auth.log
+          - /var/log/dpkg.log
+          - /var/log/syslog
+          - /var/log/messages
+          - /var/log/secure
+          - /var/log/yum.log
+          - /var/log/alternatives.log
+        exclude:
+          # Exclude logs from all containers named nr-k8s-otel-collector and otelcontribcol
+          - /var/log/pods/*/nr-k8s-otel-collector/*.log
+          - /var/log/pods/*/otelcontribcol/*.log
```

Comment:
These should be variable depending on chart release name, right?

Can this change depending on which CRI is in use?

Reply:
we actually ended up hardcoding the container name, so I updated these to reflect the hardcoded container name

thanks for the headsup on the filelog paths changing based on CRI. I added the Filelog paths for the 3 major CRI, containerd, cri-o, and docker; i figured that should be good for now, couldn't figure out how to generate the filelog paths dynamically based on the CRI, but tested and the collector won't break if the filepaths listed aren't discovered, it'll just log a message and move on w/ the filepaths it can discover.

Reply:
I am marking this al solved. I might open a new comment on a second review.